File: cash-out-gaming/app/models/Game.py
from app import db

class Game(db.Model):
    id = db.Column(db.Integer, primary_key=True)
    title = db.Column(db.String(64), nullable=False)
    description = db.Column(db.String(120), nullable=False)
    price = db.Column(db.Float, nullable=False)

    def __repr__(self):
        return f"Game('{self.title}', '{self.description}')"

File: cash-out-gaming/app/models/Game.js
const mongoose = require('mongoose');
const Schema = mongoose.Schema;

const gameSchema = new Schema({
  name: String,
  description: String,
  title: String,
  image: String,
  price: Number,
  category: String,
  genre: String,
  score: Number,
  rank: Number,
  platform: String,
  releaseDate: Date,
  rating: Number,
  reviews: [{ type: mongoose.Schema.Types.ObjectId, ref: 'Review' }],
  userId: { type: mongoose.Schema.Types.ObjectId, ref: 'User' }
});

const Game = mongoose.model('Game', gameSchema);

module.exports = Game;